src/main/resources/application.properties

```diff
@@ -5,3 +5,5 @@ spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
 
 spring.jpa.hibernate.ddl-auto=validate
 spring.jpa.show-sql=true
+jwt.expiration=3000000000
+jwt.secret=qwertyuioplkjhgfdsazxcvbnm1234567890
```

Comment:
It's better to get sensitive properties from system properties and not pushing it to git.